templates/expo-template-bare-minimum/android/app/src/main/java/com/helloworld/MainApplication.java

```diff
@@ -27,7 +27,8 @@
 public class MainApplication extends Application implements ReactApplication {
   private final ReactModuleRegistryProvider mModuleRegistryProvider = new ReactModuleRegistryProvider(
     new BasePackageList().getPackageList(),
-    Arrays.<SingletonModule>asList()
+    Arrays.<SingletonModule>asList(),
+    "INJECT_APP_ID_HERE"
```

Comment:
If we moved this property to inside `mReactNativeHost` we could use `getJSMainModuleName` here?